### ISSUE TITLE
Change the Title of a File

### ISSUE THREAD

**Issue by supercasey** [USGS-R/sbtools#230]
I'm uploading files to different pages within SB.  I'd like to be able to include a "Title" (not the file name) for the files I upload so that when users land on the page they have an idea what each file represents.  See the "hello" in the attached screenshot.  Is there a way to do this in the SB package?   

![image](https://cloud.githubusercontent.com/assets/3651871/23216678/6bc9b16a-f8dc-11e6-88dc-ae2d2dcff25d.png)

**Comment by jkreft-usgs** [comment]
looks like adding a title is on page 395 of the paper describing use of sbtools item_create:

![image](https://cloud.githubusercontent.com/assets/8128936/23220917/12bd359e-f8e9-11e6-9bdb-75793b4dfdab.png)

The whole paper is available [here.](https://journal.r-project.org/archive/2016-1/winslow-chamberlain-appling-etal.pdf)

**Comment by supercasey** [comment]
That is for adding an item as opposed to a file.  I think the process for creating an item is different than for adding a file.

**Comment by jread-usgs** [comment]
you can kind of hack this in if you need to, but I would be careful and wouldn't advise doing it. Perhaps we need to expose the file title in the "append_files" function. I don't think that was a field originally.  

The `files` field in the item is a list in R (is SB json otherwise) and you can modify it and then update the item. 

```r
item$files[[1]]$title <- "version_3_sbtools_update"
item_update(my.item, info = list(files = item$files)
```
and that works. 
![image](https://cloud.githubusercontent.com/assets/2349007/23236331/6c84ef06-f91e-11e6-9be5-76c315ca1a8f.png)

**Comment by supercasey** [comment]
Great, thanks for the help.

**Comment by jread-usgs** [comment]
@supercasey is that helpful? Also, can you give us an idea of how much of a priority this is for you and others?

**Comment by supercasey** [comment]
Yes, that was very helpful, I was able to loop through 116 separate pages and title 4-8 files each.  I would think other users that need to upload large numbers of files would like the functionality, but it may be as simple as placing your proposed fix as an example in the item_update function, there isn't much documentation in the package or in the journal article as to how to use the info argument to update metadata.
